components/automate-chef-io/content/docs/ldap.md

```diff
@@ -1,82 +1,308 @@
 +++
```

Comment:
This file name should be `LDAP.md` for consistency with SAML.md.

Reply:
In this case, all of the files should be lowercase, but good catch! I will make sure this is consistent.